Frontend/ui-driver/src/Flow.purs

```diff
@@ -449,6 +451,19 @@ updateDriverVersion dbClientVersion dbBundleVersion = do
     else pure unit
   else pure unit
 
+data FCMToken = FCMToken String
```

Comment:
Can we rewrite this function to just return the `String`. Instead of `FCMToken String`

Reply:
It given error while unwrapping so i kept data FCMTOken